proto/pdpb2.proto

```diff
@@ -0,0 +1,298 @@
+syntax = "proto3";
+package pdpb2;
+
+import "metapb.proto";
+import "eraftpb.proto";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service PD {
+    rpc Tso(TsoRequest) returns (TsoResponse) {}
+
+    rpc Bootstrap(BootstrapRequest) returns (BootstrapResponse) {}
+
+    // TODO: Carry it by rpc Bootstrap? Since a BOOTSTRAPPED Error is already
+    // definded in ResponseHeader.
+    rpc IsBootstrapped(IsBootstrappedRequest) returns (IsBootstrappedRequest) {}
+
+    rpc AllocId(AllocIdRequest) returns (AllocIdResponse) {}
+
+    rpc GetStore(GetStoreRequest) returns (GetStoreResponse) {}
+
+    rpc PutStore(PutStoreRequest) returns (PutStoreResponse) {}
+
+    rpc StoreHeartbeat(StoreHeartbeatRequest) returns (StoreHeartbeatResponse) {}
+
+    rpc AskSplit(AskSplitRequest) returns (AskSplitResponse) {}
+
+    rpc ReportSplit(ReportSplitRequest) returns (ReportSplitResponse) {}
+
+    rpc GetRegion(GetRegionRequest) returns (GetStoreResponse) {}
+
+    // TODO: Carry it by rpc GetRegion?
+    // Just a matter of a new filed in GetRegionRequest.
+    rpc GetRegionByID(GetRegionByIDRequest) returns (GetRegionResponse) {}
+
+    rpc RegionHeartbeat(RegionHeartbeatRequest) returns (RegionHeartbeatResponse) {}
+
+    rpc GetClusterConfig(GetClusterConfigRequest) returns (GetClusterConfigResponse) {}
+
+    rpc PutClusterConfig(PutClusterConfigRequest) returns (PutClusterConfigResponse) {}
+
+    rpc GetPDMembers(GetPDMembersRequest) returns (GetPDMembersResponse) {}
+}
+
+message RequestHeader {
+    // 16 bytes, to distinguish request.
+    bytes uuid = 1;
```

Comment:
I don't see any reason for keepping it, and it seems that neither TiKV nor PD uses it to distinguish request at all.
https://github.com/pingcap/tikv/blob/85fa2cad573585a850f813666442128f53bb360e/src/pd/protocol.rs#L189
https://github.com/pingcap/pd/blob/550ba5e70d473762039ca7b432bf955a885237b4/server/conn.go#L155